app/assets/stylesheets/darkswarm/_shop-product-rows.scss

```diff
@@ -105,6 +107,7 @@
           }
 
           .product-producer {
+            color: #888;
```

Comment:
I think colors should always be named variables.
In this case I think this can go to
app/assets/stylesheets/darkswarm/branding.scss
and be called $grey-550: #888;

Reply:
I was wondering about that. I will add it. :-)